test/over_react/util/handler_chain_util_test.dart

```diff
@@ -158,7 +160,7 @@ main() {
             test('calls all functions in order', () {
               var calls = [];
 
-              var functions = new List.generate(5, (index) {
+              List<Function> functions = new List.generate(5, (index) {
```

Comment:
@greglittlefield-wf @evanweible-wf when I do that, I get the following test failures:

```
00:11 +887 -1: [Dartium Content Shell] test/over_react_test.dart: HandlerChainUtil generic chaining: CallbackUtil0Arg chainFromList() returns a function of arity 0 that calls all functions in order
      type 'List<Function>' is not a subtype of type 'List<Callback0Arg>' of 'callbacks' where
        List is from dart:core
        Function is from dart:core
        List is from dart:core

      package:over_react/src/util/handler_chain_util.dart 212:41  CallbackUtil.chainFromList
      over_react/util/handler_chain_util_test.dart 167:42         main.<fn>.<fn>.sharedTests.<fn>.<fn>.<fn>
```